bundles/binding/org.openhab.binding.novelanheatpump/README.md

```diff
@@ -46,6 +46,7 @@ where `<eventType>` is one of the following values:
 | `temperature_servicewater_reference` | Number | the reference temperature of the servicewater |
 | `temperature_servicewater` | Number | the temperature of the servicewater |
 | `state` | String | contains the time of the state and the state; Possible states are error, running, stoped, defrosting |
+| `simple_state` | String | contains only the short statename; Possible states are error, running, stoped, defrosting |
```

Comment:
"stopped". Please correct also in the previous line.

Reply:
Sorry, I don't understand your comment. What do you mean?
The previous line is the original state that hasn't changed. What should changed there?

Reply:
My point was that "stoped" is incorrect, and needs to be changed to "stopped".
I was requesting you do this for both the line you added and the line above it.